gdx/src/com/badlogic/gdx/math/GeometryUtils.java

```diff
@@ -222,7 +222,7 @@ static public float polygonArea (float[] polygon, int offset, int count) {
 			x1 = x2;
 			y1 = y2;
 		}
-		return area * 0.5f;
+		return Math.abs(area * 0.5f);
```

Comment:
If this was `Math.abs(area) * 0.5f`, it would better match `triangleArea()`. But I don't believe the order makes any real difference to anything.